### DISCUSSION TITLE
[Simulation question]: Segmentation Fault in Running Nucleus DNA Damage Scorer

### DISCUSSION THREAD

**Discussion by Geoffvinc** [topas-nbio/TOPAS-nBio#32] [Simulation-question]
### Simulation question

Hi, I am using the Nucleus DNA Damage scorer example (`TOPAS-nBio/examples/scorers/SBSDamageToDNANucleus/ExampleNucleusDNADamage.txt`). 

I encountered segmentation fault when running the example with various energy. I tried changing the seed in the script, yet I still encountered the same error.

I troubleshooted the error, and I found out that the segmentation fault is caused by the scorer section in the example. Does anyone has any idea on solving the issue?

Thank you for your time.

### TOPAS-nBio category

Scoring

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by nknanfeng** [comment]
> ### Simulation question
> Hi, I am using the Nucleus DNA Damage scorer example (`TOPAS-nBio/examples/scorers/SBSDamageToDNANucleus/ExampleNucleusDNADamage.txt`).
> 
> I encountered segmentation fault when running the example with various energy. I tried changing the seed in the script, yet I still encountered the same error.
> 
> I troubleshooted the error, and I found out that the segmentation fault is caused by the scorer section in the example. Does anyone has any idea on solving the issue?
> 
> Thank you for your time.
> 
> ### TOPAS-nBio category
> Scoring
> 
> ### Code of Conduct
> * [x]  I agree to follow this project's Code of Conduct
> 
> ### Attestation
> * [x]  I testify that I have fully read the bug reporting guideline beforehand.

Hi, Geoffvinc
I hope this email finds you well.

I would like to inquire if you have more detailed information regarding the segment fault. Could you kindly provide the context surrounding the error, such as any logs or additional details that might help in identifying the root cause?


Best regards,
nanfeng

**Reply by Geoffvinc** [reply]
Hi Nan Feng,
The error return the log below
`line 7:  8433 Segmentation fault      (core dumped) $HOME/Applications/TOPAS/OpenTOPAS-install/bin/topas $1`
The error log doesn't provide any detail regarding the segmentation fault, therefore I think it can be exhausting to identify the main cause of the error.
As of now, I only identified that the error is caused in the scorer function. The script runs well if I omit the scorer section.
Hope this information helps. Thank you!

Regards,
Geoffvinc
